mislabeled/detect/detectors.py

```diff
@@ -69,6 +70,26 @@ def __init__(self, base_model):
         )
 
 
+class ALOODetector(ModelProbingDetector):
```

Comment:
Wouldn't it be more approriate to name it something like One Newton-Step approximation to Cook's distance?

If you are positive that "ALOO" is actually already used in the literature I am perfectly fine with using it, but in general I am reluctant to introduce new names for existing techniques.